bmi.rb

```diff
@@ -0,0 +1,7 @@
+puts "Bienvenido al calculador de índice de masa corporal (IMC), o BMI por sus siglas en inglés."
+puts "Por favor ingresa tu peso"
+weight = gets.chomp.to_i
+puts "Por favor ingresa tu altura"
+height = gets.chomp.to_f
+imcRound =  (weight / (height**2)).round
```

Comment:
in ruby ​​the convention for variables and methods is snake_case
```suggestion
imc_round =  (weight / (height ** 2)).round
```